environs/manual/sshprovisioner/package_test.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2013 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// +build linux
```

Comment:
Can you please have it build on all, but just t.Skip in the Test function? That way it's more obvious why the tests aren't running.